pvlib/ivtools.py

```diff
@@ -262,6 +264,177 @@ def fit_sde_sandia(voltage, current, v_oc=None, i_sc=None, v_mp_i_mp=None,
+    irradiance and cell temperature.
+
+    Parameters
+    ----------
+    celltype: str, case insensitive
+        Value is one of 'monosi', 'multisi', 'polysi' or'gaas'.
+        Others like 'cis', 'cigs', 'cdte', 'amorphous' are not implemented yet
+    v_mp: numeric
+        Module voltage at the maximum-power point at std conditions in V.
+    i_mp: numeric
+        Module current at the maximum-power point at std conditions in A.
+    v_oc: numeric
+        Open-circuit voltage at std conditions in V.
+    i_sc: numeric
+        Short-circuit current at std conditions in A.
+    alpha_sc: numeric
+        The short-circuit current (i_sc) temperature coefficient of the
+        module in units of %/K. It is converted in A/K for the computing
+        process.
+    beta_voc: numeric
+        The open-circuit voltage (v_oc) temperature coefficient of the
+        module in units of %/K. It is converted in V/K for the computing
+        process.
+    cells_in_series: numeric
+        Number of cell in the module.
+        Optional input, but helps to insure the convergence of the computing.
+    temp_ref: numeric, default 25
+        Reference temperature condition [C]
+    irrad_ref: numeric, default 1000
+        Reference irradiance condition [W/m2]
+
+    Returns
+    -------
+    Dictionnary with the following elements:
+
+    * 'I_L_ref': numeric
+        Light-generated current in amperes at std conditions.
+    * 'I_o_ref': numeric
+        Diode saturation curent in amperes at std conditions
+    * 'R_s': numeric
+        Series resistance in ohms. Note that '_ref' is not mentionned
+        in the name because this resistance is not sensible to the
+        conditions of test.
+    * 'R_sh_ref': numeric
+        Shunt resistance in ohms at std conditions.
+    * 'a_ref' : numeric
+        Modified ideality factor at std conditions.
+        The product of the usual diode ideality factor (n, unitless),
+        number of cells in series (Ns), and cell thermal voltage at
+        specified effective irradiance and cell temperature.
+    * 'alpha_sc': numeric
+        Caution!: Different from the input because of the unit.
+        The short-circuit current (i_sc) temperature coefficient of the
+        module in units of A/K.
+    * 'EgRef': numeric
+        Energy of bandgap of semi-conductor used (depending on celltype) [J]
+    * 'dEgdT': numeric
+        Variation of bandgap according to temperature [J/K]
+    * 'irrad_ref': numeric
+        Reference irradiance condition [W/m2]
+    * 'temp_ref': numeric
+        Reference temperature condition [C]
+
+    References
+    ----------
+    [1] W. De Soto et al., "Improvement and validation of a model for
+    photovoltaic array performance", Solar Energy, vol 80, pp. 78-88,
+    2006.
+
+    [2] John A Dufﬁe ,William A Beckman, "Solar Engineering of Thermal
+    Processes", Wiley, 2013
+    """
+    # Constants
+    k = constants.Boltzmann  # in J/K, or 8.617e-5 eV/K
```

Comment:
`k = constants.value('Boltzmann constant in eV/K')` for the Boltzmann constant in eV/K.